modules/example/src/smithy4s/example/Age.scala

```diff
@@ -0,0 +1,13 @@
+package smithy4s.example
```

Comment:
This obviously shouldn't be generated, but I didn't do any of the renderer work yet